wiki/Silence/en.md

```diff
@@ -10,17 +6,35 @@ stub: true
 
 Users who are silenced by moderators cannot do the following:
 
-- Edit their userpage
-- Change their avatar/profile details <!-- unconfirmed on new site, but was a thing on old -->
-- Submit [beatmaps](/wiki/Beatmap)
-- Send/receive private messages
+- Use [chat](/wiki/Chat_Console), both in-game and on the website
+- Send and receive private messages
```

Comment:
Is chat and pm the same? I would merge into one if that is the case.
```suggestion
- Use [chat](/wiki/Chat_Console) (public and private), both in-game and on the website
```

Reply:
yes